README.md

```diff
@@ -13,15 +13,19 @@ git clone git@github.com:maas/maas-ansible-playbook
 ## Setup
 This playbook has several main roles, each has a corresponding group to assign hosts to. They are as follows:
 
-- **maas_postgres_primary**: This role and group is for a host to be configured as an initial Postgres primary instance for the MAAS stack. This role is required.
+- **maas_pacemaker**: This role and group are for a set of hosts to be configured as a pacemaker cluster to manage HA Postgres. It is recommended that `maas_corosync` is a child group of this group. It is also recommended that these are the same hosts as the postgres group.
```

Comment:
This role and group are for a set of hosts to be configured as a Pacemaker cluster to manage HA Postgres. It is recommended that `maas_corosync` is a child group of this group. It is also recommended that these are the same hosts as the Postgres group.